tests/unit/parsers/test.manifestjson.js

```diff
@@ -4375,17 +4446,21 @@ describe('ManifestJSONParser', () => {
     it('supports an object compat data for Android', () => {
       const support = {
         firefox_android: {
-          version_added: '85',
+          version_added: '114',
         },
       };
 
-      _checkKeySupport({ support, minVersion: 10 });
+      _checkKeySupport({
+        support,
+        minFirefoxVersion: 42,
+        minAndroidVersion: 113,
+      });
 
       expect(addonLinter.collector.notices).toHaveLength(1);
       expect(addonLinter.collector.notices[0]).toEqual(
         expect.objectContaining({
           description: oneLine`"strict_min_version" requires Firefox for
-            Android 48.0.0, which was released before version 85 introduced
+            Android 113, which was released before version 114 introduced
```

Comment:
Well, I guess what I am trying to say is: could we have _new_ coverage instead of updating existing tests?

Reply:
The old tests were relying on behavior that no longer exists (using the same version for both) so they had to be updated, and I think that covers everything now. I'm happy to add more but I'm not sure what to add.